app/views/compute_resources/form/_ovirt.html.erb

```diff
@@ -1,6 +1,7 @@
 <%= text_f f, :url, :size => "col-md-8", :help_block => _("e.g. https://ovirt.example.com/api"), :help_inline => documentation_button('5.2.7oVirt/RHEVNotes') %>
 <%= text_f f, :user, :help_block => _("e.g. admin@internal") %>
 <%= password_f f, :password, :unset => unset_password? %>
+<%= checkbox_f f, :filtered_api, :help_inline => _("If you don't have admin permissions you can use slower filtered API."), :label => _('User level API') %>
```

Comment:
"Faster filtering for priviledged accounts" does not sound clear to me, not sure how it suggests anything about pagination and searching not being possible. I'm happy to change the inline help text but (anyone) please suggest the full sentence. Or more sentences that we'd hide in popover. I think the existing text is best so far. Maybe "User level API" -> "User account" label change would be enough?